src/main/java/core/basesyntax/figure/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figure;
+
+public class RightTriangle extends AbstractFigure implements Figure {
```

Comment:
The class RightTriangle should not implement Figure directly since it is already extending AbstractFigure, which presumably implements Figure. Remove the direct implementation to avoid redundancy.